docs/nidigital/class.rst

```diff
@@ -1243,60 +1243,47 @@ frequency_counter_measure_frequency
 
 
 
-get_channel_name
-----------------
+get_channel_names
+-----------------
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: get_channel_name(index)
-
-            TBD
-
-            
-
-
-
-            :param index:
-
-
-                
-
-
-            :type index: int
-
-            :rtype: str
-            :return:
-
-
-                    
-
+    .. py:method:: get_channel_names(indices)
 
+            Returns a list of channel names for given channel indices.
 
-get_channel_name_from_string
-----------------------------
+            This is useful in multi-instrument sessions, where channels are expected to be
+            referenced by their fully-qualified names, for example, PXI1Slot3/0.
 
-    .. py:currentmodule:: nidigital.Session
-
-    .. py:method:: get_channel_name_from_string(index)
+            
 
-            TBD
 
-            
 
+            :param indices:
 
 
-            :param index:
+                Specifies indices for the channels in the session.
+                Valid values are from zero to the total number of channels in the session minus one.
+                The following types and formats are supported:
+                  - int - for example, 0
```

Comment:
I think you should use colon rather than comma, and the word "for" isn't needed.

```
- int - example: 0
- Basic sequence - example: [0, 1, 2], [0, range(2, 4)]
- str - example: "0, 2, 3, 1", "0-3", "0:3"
```

so that all lines have mu

Reply:
Fixed